### PR TITLE
 OFS-15: Added models and services shell

### PR DESCRIPTION
**Pull request by dborowiecki**
TICKET: [OFS-15](https://openimis.atlassian.net/browse/OFS-15)

### CODE REVIEW

File: contribution_plan/models.py [hunk 1]
```diff
@@ -1,3 +1,114 @@
...
+    user_updated = models.ForeignKey(core_models.InteractiveUser, db_column="UserUpdatedUUID",
+                                     related_name="%(class)s_UpdatedUUID", on_delete=models.deletion.DO_NOTHING)
+    user_created = models.ForeignKey(core_models.InteractiveUser, db_column="UserCreatedUUID",
+                                     related_name="%(class)s_CreatedUUID", on_delete=models.deletion.DO_NOTHING)
+
+    date_valid_from = fields.DateTimeField(db_column="DateValidFrom")
+    date_valid_to = fields.DateTimeField("DateValidTo")
+
+    class Meta:
+        managed = False
```
Comment (delcroip) [inline contribution_plan/models.py:39]: Why not turning on Django management ?

File: contribution_plan/services.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        pass
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.__dict__ == other.__dict__
+
+    def replace_propagation(self, policy_holder):
+        pass
+
+    def remove_propagation(self, policy_holder):
+        pass
```
Comment (delcroip) [inline contribution_plan/services.py:18]: Could you add update and create ?

File: contribution_plan/services.py [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+@core.comparable
+class ContributionPlanBundle(object):
+
+    def __init__(self, contribution_plan):
+        self.contribution_plan = contribution_plan
+        pass
+
+    def __eq__(self, other):
+        return isinstance(other, self.__class__) and self.__dict__ == other.__dict__
```
Comment (delcroip) [inline contribution_plan/services.py:40]: Could you add update, create, remove and replace ?

**Review by delcroip — CHANGES_REQUESTED** [review]

File: contribution_plan/models.py (hunk 1 above)
Comment (dborowiecki) [reply contribution_plan/models.py]: It was not necessary, I've restored default management

File: contribution_plan/services.py (hunk 2 above)
Comment (dborowiecki) [reply contribution_plan/services.py:18]: Added

File: contribution_plan/services.py (hunk 3 above)
Comment (dborowiecki) [reply contribution_plan/services.py:46]: Added

**Comment by dborowiecki** [comment]
@delcroip I've made changes to PR, can you take a look?

**Review by delcroip — APPROVED** [review]
OK for me

**Pull request merged**